pom.xml

```diff
@@ -891,6 +905,41 @@ under the License.
 
 	<build>
 		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-dependency-plugin</artifactId>
+				<version>3.0.2</version>
```

Comment:
Redundant from the `<pluginManagement>` section.

Reply:
will remove.